### DISCUSSION TITLE
Cannot use import statement outside a module

### DISCUSSION THREAD

**Discussion by i-afzal** [quasarframework/quasar#14140] [CLI - SSR mode]
This is the generated package.json file by Quasar cli for ssr mode
![image](https://user-images.githubusercontent.com/109411131/183001428-f39cd587-6adc-4dab-b59a-459a056260d5.png)

When I try to run yarn start, I am getting this error

![image](https://user-images.githubusercontent.com/109411131/183001636-891864b4-2721-435b-b205-a6bf6d032e6f.png)

Please help

**Comment by pdanpdan** [comment]
Did you run yarn or npm install in that folder before trying to execute?

**Reply by i-afzal** [reply]
tried with both npm install and yarn install

**Reply by i-afzal** [reply]
In one of my components, I am directly referencing `import { useFormAttrs } from 'quasar/src/composables/private/use-form.js` and this is causing the above issue only in ssr build

I require this to override default behaviour of QSlider by defining my own custom component.

Any ideas, how to resolve this ?

**Reply by stefanvanherwijnen** [reply]
You can try adding `"type": "module" ` to your ssr server's package json. However, this might break other things.

**Reply by i-afzal** [reply]
I tried it, but it broke other things. This issue is only with Vite build
